dv/verilator/memutil/cpp/verilator_memutil.cc

```diff
@@ -228,9 +228,13 @@ bool VerilatorMemUtil::IsFileReadable(std::string filepath) const {
 bool VerilatorMemUtil::ElfFileToBinary(const std::string &filepath,
                                        uint8_t **data,
                                        size_t &len_bytes) const {
-  bool retval;
-  std::list<BufferDesc> buffers;
-  size_t offset = 0;
+  uint8_t *buf;
+  bool retval, any = false;
+  GElf_Phdr phdr;
+  GElf_Addr high = 0, low = (GElf_Addr)-1;
```

Comment:
can you split this into two lines, and run clang-format on the whole change to fix the whitespace?

Reply:
Fixed